testing/trino-product-tests/src/main/java/io/trino/tests/product/TestSystemConnector.java

```diff
@@ -108,4 +121,102 @@ public void selectMetadataCatalogs()
                 .hasColumns(VARCHAR, VARCHAR)
                 .hasAnyRows();
     }
+
+    @Test(groups = {SYSTEM_CONNECTOR, JDBC})
+    public void selectMetadataMaterializedViews()
```

Comment:
This doesn't need to be product test.
You can add the tests under `BaseConnectorTest#testMaterializedView`

Reply:
I moved the test under `BaseIcebergConnectorTest` since only Iceberg supports MV creation.

Reply:
The test should go under `BaseConnectorTest#testMaterializedView` because we want to have connector agnostic test where possible. If some other connector implements MVs, we want it to automatically inherit the existing tests, notice that we already have similar tests around metadata listing in  `BaseConnectorTest#testMaterializedView` even though iceberg is the only implementation.
If some MV test really depends on iceberg specific behaviour, then it needs to go into TestIcebergMaterializedViews

Reply:
^ ping

Reply:
Gotcha, I'll move it up into `BaseConnectorTest` in that case 👍